test/utest.cpp

```diff
@@ -174,6 +173,25 @@ TEST(PluginlibTest, createManagedInstanceAndUnloadLibrary)
   ROS_INFO( "Done." );
 }
 
+TEST(PluginlibTest, brokenXML)
+{
+  try
+  {
+    pluginlib::ClassLoader<test_base::Fubar> test_loader("pluginlib", "test_base::Fubar", "plugin_test");
+  }
+  catch(pluginlib::ClassLoaderException& ex)
+  {
+    SUCCEED();
+    return;
+  }
+  //catch(...)
+  //{
+  //  FAIL() << "Uncaught exception";
+  //}
```

Comment:
these commented lines can be removed given that the test exits on line 185 if the expected exception is thrown